### PR TITLE
Allow ip command

### PR DESCRIPTION
**Pull request by Westwooo**

### CODE REVIEW

File: src/cli/allow_ip.rs [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            .category(Category::Custom("couchbase".to_string()))
+            .optional(
+                "address",
+                SyntaxShape::String,
+                "ip address to allow access to the cluster",
+            )
+    }
+
+    fn usage(&self) -> &str {
+        "Whitelists an IP address on a Capella cluster"
```
Comment (chvck) [inline src/cli/allow_ip.rs:39]: "Adds an IP address to the allowlist on a Capella cluster" maybe? I think we've moved away from using whitelist?

File: src/cli/allow_ip.rs [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+            deadline,
+            ctrl_c,
+        )
+        .map_err(|e| client_error_to_shell_error(e, span))?;
+
+    Ok(PipelineData::empty())
+}
+
+fn format_ip_address(ip_address: String) -> String {
+    if !ip_address.ends_with("/32") {
```
Comment (chvck) [inline src/cli/allow_ip.rs:142]: ip address can't be any other format can it? Just checking that we aren't blocking some usage.

Comment (Westwooo) [reply src/cli/allow_ip.rs]: The addresses required a subnet mask, and if not given the UI defaults to try and use '/32', so I've changed this to behave the same.

File: src/cli/allow_ip.rs (hunk 1 above)
Comment (Westwooo) [reply src/cli/allow_ip.rs]: Done!

**Comment by ldoguin** [comment]
Fails in loop if the current project is not set.

**Comment by Westwooo** [comment]
> Fails in loop if the current project is not set.

That's expected, unless passing in a projects with the --project flag

**Review by Matt-Woz — APPROVED** [review]
LGTM 👍

**Pull request merged**